docker/run_crontabber.sh

```diff
@@ -4,18 +4,15 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-# Runs crontabber.
+# Runs cronrun (used to be called crontabber).
 
 set -e
 
-# Run crontabber sleeping 5 minutes between runs
+# Run cronrun sleeping 5 minutes between runs
 while true
 do
-    # NOTE(willkg): We don't want crontabber to exit weird and then have that
+    # NOTE(willkg): We don't want cronrun to exit weird and then have that
     # kill the container, but this is a lousy thing to do.
-    ./socorro-cmd crontabber || true
```

Comment:
Because I removed the last job, I figured I'd remove crontabber from the script, too. I don't know what crontabber would do if it has no jobs, but I didn't feel like finding out since it's going to be removed in a followup PR anyhow.